app/controllers/application_controller.rb

```diff
@@ -0,0 +1,18 @@
+require 'pry'
+require './config/environment'
+
+class ApplicationController < Sinatra::Base
+
+  get "/" do
```

Comment:
Do we want to leave the route as a root path (/)? Curious if we would prefer a more explicit route like `/weather`.

Reply:
yes, I think that this will need to be updated..

Reply:
Definitely! This was more of a placeholder to get the API up and running. Good call!

Reply:
That has been updated